phocoa/framework/WFAuthorization.php

```diff
@@ -467,7 +479,14 @@ function logout()
     {
         $this->init();
 
-        // clear remember me cookie
+        $this->clearRememberMe();
+    }
+
+    /**
+     * Clear the remember me cookie
+     */
+    function clearRememberMe()
+    {
         $options = $this->rememberMeOptions();
         setcookie($options[self::REMEMBER_ME_OPT_NAME], '', strtotime('-1 year'), $options[self::REMEMBER_ME_OPT_PATH], $options[self::REMEMBER_ME_OPT_DOMAIN]);
```

Comment:
For safety you should also `unset ($_COOKIE[$options[self::REMEMBER_ME_OPT_NAME]]);` that way if execution did continue and try to access the cookie it would die.